releases/release-1.26/README.md

```diff
@@ -60,35 +61,35 @@ The 1.26 release cycle is as follows:
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings) | Lead | Monday 7th November 2022                                                                                                          | week 10  | [1.26-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]**                          | Lead | Monday 7th November 2022                                                                                                          | week 10  | |
+| Brace Yourself, Code Freeze is Coming                         | Comms / Bug Triage | Monday 7th November 2022                                                                                            | week 10  | |
+| **Begin Feature blog freeze**                                 | Comms Lead | 01:00 UTC Wednesday 9th November 2022 / 18:00 PDT Tuesday 8th November 2022                                                 | week 10  | |
+| Release retrospective part 1                                  | Community | 17:00 UTC Wednesday 9th November 2022 / 10:00 PDT Wednesday 9th November 2022                                                | week 10  | |
+| Burndown Meetings daily                                       | Lead | Monday 14st November 2022                                                                                                         | week 11  | |
+| **Begin [Code Freeze]**                                       | Branch Manager | 01:00 UTC Wednesday 16th November 2022 / 02:00 BST Wednesday 16th November 2022 / 18:00 PDT Tuesday 15th November 2022  | week 11  | |
+| 1.26.0-beta.0 released                                        | Branch Manager | Thursday 17th November 2022                                                                                             | week 11  | |
+| Docs deadline — Open placeholder PRs                          | Docs Lead | Thursday 17th November 2022                                                                                                  | week 11  | |
+| Deprecations and Removals blog published                      | Comms | Thursday 17th November 2022                                                                                                      | week 11  | |
+| **[Test Freeze]**                                             | Branch Manager | 01:00 UTC Wednesday 23rd November 2022 / 02:00 BST Wednesday 23rd November 2022 / 18:00 PDT Tuesday 22nd November 2022  | week 12  | |
+| Docs deadline — PRs ready for review                          | Docs Lead | Tuesday 22nd November 2022                                                                                                   | week 12  | |
+| 1.26.0-rc.0 released                                          | Branch Manager | Tuesday 22nd November 2022                                                                                              | week 12  | |
```

Comment:
Do we want to time rc0 and rc1 such that there is a plausible way someone could try it, report an issue, and the issue could be resolved before rc1? If so, I doubt many consumers will pick up rc.0 starting on Wednesday on a US holiday week (I'm not sure when during the day on 11/22 the release would happen). If they do pick it up, feedback won't be likely until a couple days later (I wouldn't expect feedback before ~Thursday), at which point many folks will be unavailable to drive a fix in time for rc1 on 11/29.

Reply:
hmm, yes that could be a problem. There is little room to play with.. 

Options I can think of:
1. move rc.0 by a day to Monday 11/21 (move inside the schedule)
2. move feature blog freeze, code freeze + rc.0 up by a week

Opinions?

Reply:
> move feature blog freeze, code freeze + rc.0 up by a week

I like this, especially since code freeze *always* has a few days of clean up work that escape anyway

Reply:
> move feature blog freeze, code freeze + rc.0 up by a week

👍

Reply:
I'm pro moving up a week mostly bc it reflects and accommodates reality a bit more. I don't think many USians will pick up, test and formulate feedback about the new release on Tuesday/Wednesday.  @liggitt 's point about no room for an rc.2 is also a good one.